src/types/user.types.ts

```diff
@@ -1,10 +1,10 @@
 export interface UserTypes {
-    id: number;
+    Id: number;
     Username: string;
-    email: string;
-    password: string;
-    isVerified: boolean;
-    isOnboarding: boolean;
-    createdAt: string;
-    updatedAt: string;
+    email?: string;
+    password?: string;
+    isVerified?: boolean;
+    isOnboarding?: boolean;
+    createdAt?: string;
+    updatedAt?: string;
```

Comment:
Make the first letter in capital letters.

Reply:
done